### PR TITLE
Detox: Android - fix smoke server login test

### PR DESCRIPTION
**Pull request by metanerd**
#### Summary

Screenshot from 2023-02-23 18-48-22

#### Checklist

- [ ] Added or updated unit tests (required for all new features)
- [ ] Has UI changes
- [ ] Includes text changes and localization file updates
- [ ] Have tested against the 5 core themes to ensure consistency between them.

#### Device Information
This PR was tested on: 

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by larkox — APPROVED** [review]

File: detox/e2e/test/smoke_test/server_login.e2e.ts [hunk 1]
```diff
@@ -79,11 +79,11 @@ describe('Smoke Test - Server Login', () => {
 
         // # Go back to first server, open server list screen, swipe left on second server and tap on logout option
         await ServerListScreen.open();
-        await ServerListScreen.serverListScreen.swipe('up');
+        await ServerListScreen.serverListTitle.swipe('up', 'fast', 0.1, 0.5, 0.3);
```
Comment (saturninoabril) [inline detox/e2e/test/smoke_test/server_login.e2e.ts:82]: The change here and the other one failed on me in iOS due to bottom sheet not being pulled up fully causing the "Log Out" button to be partially covered (per Detox visibility check).
Screenshot 2023-02-27 at 6 11 45 PM
Screenshot 2023-02-27 at 6 11 29 PM

However, the original code passed in iOS.
Screenshot 2023-02-27 at 6 25 31 PM

How about change the swipe up action to Android only?

**Review by saturninoabril — CHANGES_REQUESTED** [review]

**Review by saturninoabril — APPROVED** [review]
Thanks @metanerd, LGTM.

**Comment by larkox** [comment]
@metanerd it seems you have some lint errors

**Pull request merged**